app/assets/stylesheets/rentals/rental_schedule.css

```diff
@@ -17,7 +17,8 @@ table, th, td {
 }
 
 .simple-calendar table td {
-  height: 100px !important;
+  height: 123px !important;
+  width : 163px !important;
```

Comment:
It does not because the standard table td is even higher in the stack, the !important 1 ups the default table css.